docs/source/get-started/configuration-guide.rst

```diff
@@ -182,9 +182,12 @@ Debugging
       * Use bounds checking - will increase runtime
       * ``OFF``
 
-    * * ``Kokkos_ENABLE_DEBUG_DUALVIEW_MODIFY_CHECK``
+    * * ``Kokkos_ENABLE_DEBUG_DUALVIEW_MODIFY_CHECK`` :red:`[Deprecated since 4.7]`
```

Comment:
Does the [deprecated](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) directive work here? I think it would be better as it is more canonical.

Reply:
It may but I chose to go for consistency with the rest of the page.
I would suggest we attempt to use the deprecated directive to all in a follow up PR.

Reply:
Ok fair enough!